section1/set6/exercises/07-leap-year/main.go

```diff
@@ -37,4 +43,26 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	// declare variables
+	a := os.Args
+	var n int
```

Comment:
variable should be declared as close to the place where it is used

Reply:
fixed